### PR TITLE
[WFLY-16036] Use default security domain for EJBs when no default specified

### PR DESCRIPTION
**Pull request by skyllarr**
https://issues.redhat.com/browse/WFLY-16036

### CODE REVIEW

File: ejb3/src/main/java/org/jboss/as/ejb3/component/EJBComponent.java [hunk 1]
```diff
@@ -274,7 +275,7 @@ public Principal getCallerPrincipal() {
         }
 
         // TODO Should this return a Principal when security is not activated.
```
Comment (fjuma) [inline ejb3/src/main/java/org/jboss/as/ejb3/component/EJBComponent.java:277]: Just minor, but we can remove this TODO comment now. :)

**Review by fjuma — APPROVED** [review]
Thanks @Skyllarr!

**Review by darranl — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks, @Skyllarr

File: ejb3/src/main/java/org/jboss/as/ejb3/component/EJBComponent.java (hunk 1 above)
Comment (rhusar) [reply ejb3/src/main/java/org/jboss/as/ejb3/component/EJBComponent.java:277]: Looks like this got merged with the TODO still in place :-P

Comment (Skyllarr) [reply ejb3/src/main/java/org/jboss/as/ejb3/component/EJBComponent.java:277]: @rhusar I submitted PR that removes the TODO comment https://github.com/wildfly/wildfly/pull/15238 :tada:

**Pull request merged**